debian/changelog

```diff
@@ -1,3 +1,10 @@
+matrix-synapse-py3 (1.80.0+nmu1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Don't make homeserver config file and data directory world-readable
```

Comment:
It's normal to include author info here (see https://github.com/matrix-org/synapse/blob/5a5106dc1d62e2f3e049544c0fa6c8de8be024b3/debian/changelog#L272). `dch` will do this for you normally; otherwise you can do it manually.